sema/subyt/sources.py

```
@@ -87,43 +87,56 @@ def mime_from_identifier(identifier: str) -> str:
         super().__init__()
-        self._collection_path = "."
-        self._sourcefiles = []
+        self._collection_path: Path = Path(".")
+        self._sourcefiles: list[Path] = []
 
     def __repr__(self):
         return f"{type(self).__name__}('{self._collection_path}')"
 
+    def _init_sourcefiles(self, source_paths: list[Path]):
+        self._sourcefiles = sorted(source_paths)
+        assert len(self._sourcefiles) > 0, (
```

Ruff (0.8.2)

Use of `assert` detected

(S101)